src/libs/OptionsListUtils.js

```diff
@@ -209,26 +209,35 @@ function getSearchText(report, reportName, personalDetailList, isChatRoomOrPolic
 }
 
 /**
- * If the report or the report actions have errors, return
- * CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR, otherwise an empty string.
- *
+ * Get an object of error messages keyed by microtime by combining all error objects related to the report.
  * @param {Object} report
  * @param {Object} reportActions
- * @returns {String}
+ * @returns {Object}
  */
-function getBrickRoadIndicatorStatusForReport(report, reportActions) {
+function getAllReportErrors(report, reportActions) {
     const reportErrors = report.errors || {};
     const reportErrorFields = report.errorFields || {};
     const reportID = report.reportID;
     const reportsActions = reportActions[`${ONYXKEYS.COLLECTION.REPORT_ACTIONS}${reportID}`] || {};
+    const reportActionErrors = {};
+    _.chain(reportsActions)
+        .filter(action => !_.isEmpty(action.errors))
+        .map(action => _.extend(reportActionErrors, action.errors));
```

Comment:
NAB: since we're not returning a value

```suggestion
        .each(action => _.extend(reportActionErrors, action.errors));
```

Reply:
Oh yeah good idea.